apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -499,13 +502,22 @@ static void tc_task_task_reparent(void)
 */
 static void tc_task_task_init(void)
 {
+	int ret = 0;
+
 	task_init_flag = false;
 
+	sem_init(&task_sem, 0, 0);
+
 	TC_ASSERT_EQ("task_init", ioctl(tc_get_drvfd(), TESTIOC_TASK_INIT_TEST, (unsigned long)test_task_entry), OK);
 
 	usleep(1);
```

Comment:
Do we still need this usleep even we use sem_wait?

Reply:
Could you reply this?

Reply:
Was checking on tc_success, it is working fine, will push without usleep